examples/sagemaker-pipelines-graphbolt/README.md

```diff
@@ -249,9 +250,9 @@ Next you will build and push the GraphStorm PyTorch Docker image that you'll use
 
 
 ```bash
-# Ensure Docker is installed
+# Ensure dependencies are installed
 sudo apt update
-sudo apt install -y Docker.io
+sudo apt install rsync
```

Comment:
Any reason to remove this ? rsync comes pre-installed in regular Ubuntu distributions, a minimized image might not have it.